src/azul/service/avro_pfb.py

```diff
@@ -329,6 +329,59 @@ def pfb_schema_from_replicas(replicas: Iterable[JSON]
     return keys, _avro_pfb_schema(values)
 
 
+def pfb_schema_from_anvil_schema(anvil_schema: JSON
```

Comment:
This, and the other AnVIL-specific artifacts should live in the AnVIL plugin module. I expect more plugins that file formats, so I'd rather have file-format specifics distributed over the various plugin modules than the other way around.

Reply:
Agreed at PL that ` azul.avro_pfb._avro_pfb_schema` can be made public to facilitate this move.